com.unity.render-pipelines.core/Runtime/Common/DynamicResolutionHandler.cs

```diff
@@ -274,6 +282,29 @@ public float GetCurrentScale()
             return (m_Enabled && m_CurrentCameraRequest) ? m_CurrentFraction : 1.0f;
         }
 
+        /// <summary>
+        /// Gets the resolved scale percentages.
+        /// Note: this function is pure, and has no side effects.
+        /// </summary>
+        /// <returns>The resolved drs scales (from 0 to 1, for x and y). These might change depending on what the hardware supports.</returns>
```

Comment:
In documentation we should avoid acronyms (Dynamic Resolution here instead of DRS)

Reply:
Fixed this! done